package-lock.json

```diff
@@ -42177,12 +42185,6 @@
 			"integrity": "sha1-Epai1Y/UXxmg9s4B1lcB4sc1tus=",
 			"dev": true
 		},
-		"json2php": {
```

Comment:
It would be great, if the package would get updated here (top-level `node_modules`, instead of becoming a direct dependency of the package (`node_modules` inside the package).

Reply:
Thanks! Done 👍